### DISCUSSION TITLE
Editor Problem

### DISCUSSION THREAD

**Discussion by MaroDiamond9** [ppy/osu#14525] [Q&A]
The Audio Select Is Not Functioning Well On Mobile.

**Comment by PercyDan54** [comment]
#11911

**Reply by frenzibyte** [reply]
This is 100% missing details to even decide what the issue is.

**Comment by frenzibyte** [comment]
@MaroDiamond9 Please provide further details, what's "not functioning" in the audio file selector? attach screenshots/videoes to understand what the problem is.
